python/whylogs/core/metrics/metrics.py

```diff
@@ -40,6 +40,7 @@ class MetricConfig:
     kll_k: int = 256
     fi_lg_max_k: int = 10  # 128 entries
     fi_disabled: bool = False
```

Comment:
If I were me, I'd move the metric enable/disable to a separate class (`ColumnSchema`? or maybe a separate `ColumnConfig`?) and have `MetricConfig` just hold actual parameters for `Metric`s.  It also seems like these overlap with resolvers... Is there guidance on when to `resolve` vs `MetricConfig`?  Also, `UnicodeRangeMetric` tracks string lengths in addition to range counts, so it might be unclear that `track_unicode_ranges` turns the non-rangy parts of the metric on/off. There's also the case of `MetricConfig.unicode_ranges = {}`, which will still track a single `UNKNOWN` range.

Reply:
Those refactors do sound reasonable. I vote to keep this change small and move forward with the bug fix portion here first + tests. Let's follow up with the refactor around ColumnConfig vs MetricConfig in a separate PR.